plasticos_documents_native/models/document_native.py

```
@@ -0,0 +1,169 @@
+            })
+        _logger.info(
+            "Documents verified: %s",
+            ", ".join(str(r.id) for r in self),
+        )
+
+    def action_override(self):
+        """Mark the document as overridden (requires manager group)."""
+        self.ensure_one()
+        if not self.env.user.has_group("base.group_system"):
+            from odoo.exceptions import UserError
+            raise UserError("Only administrators can override documents.")
```

Ruff (0.15.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)